app/models/financialDetails/FinancialDetail.scala

```diff
@@ -88,11 +88,16 @@ case class FinancialDetail(taxYear: String,
     }
     .filter(_.getPaymentAllocationTextInChargeSummary.isDefined)
 
-  def isMFACredit: Boolean = MfaCreditUtils.validMFACreditType(mainType)
-
-  def isCutOverCredit: Boolean = mainType.contains("ITSA Cutover Credits")
-
-  def isBalancingChargeCredit: Boolean = mainType.contains("SA Balancing Charge Credit")
+  def getCreditType: Option[String] = {
```

Comment:
pls use strong typing instead of Option[String] as this is suggested in the main story